_content/monographs/summer-mountain-timeless-landscape_wen-fong.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "Summer Mountains: The Timeless Landscapes"
+authors:
+  - Wen Fong
```

Comment:
Not strictly necessary, but by convention I usually put full names like this in quotes. (Also the publisher field below)